ktor-io/common/src/io/ktor/utils/io/core/Strings.kt

```diff
@@ -89,18 +89,14 @@ public fun Source.readTextExactCharacters(charactersCount: Int, charset: Charset
 }
 
 /**
- * Writes [text] characters in range \[[fromIndex] .. [toIndex]) with the specified [charset]
+ * Writes [text] characters in range [fromIndex] .. [toIndex]) with the specified [charset]
```

Comment:
Could you check the KDoc and mention explicitly included and excluded part of the ranges?